PriceQuoteIos/Models/Salesman.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  Salesman.swift
+//  PriceQuoteIos
+//
+//  Created by Livia Vasconcelos on 27/09/20.
+//  Copyright © 2020 Livia Vasconcelos. All rights reserved.
+//
+
+import Foundation
+import ObjectMapper
+
+struct Salesman {
+    let name: String
+    let status: SalesmanStatus
+    let phone: Phone
+    
+    func format() -> [String: Any] {
+        return [
+            "salesman": [
+                "name": name, 
+                "status": status.rawValue,
+                "phone": [
+                    ["number": phone.number,
+                    "whatsapp": phone.isWhatsappActive]
+                ]
+            ]
+        ]
+    }
```

Comment:
For this structure, I, personally, would rather use `Codable`. It is ok to work this way, but with codable you might not need to worry much with JSON nesting (only in specific cases, like repeated keys) if you work with `CodingKeys`

Reply:
Did not do it for now, because I couldn't make it work hehhe

Reply:
Fixed!